### DISCUSSION TITLE
RefreshContainer: RefreshVisualizer not visible?

### DISCUSSION THREAD

**Discussion by msneijders** [AvaloniaUI/Avalonia#14427] [Q&A]
Testing in the ControlCatalog.Android (current main branch) RefreshContainer page and running simulator:

When I do a pull-to-refresh the listbox items move downwards, but I do not see the visualizer; I remember it being a circular arrow rotating while dragging?

Is it working for others?

**Comment by maxkatz6** [comment]
cc @emmauss

**Reply by timunie** [reply]
Is any StyleInclude needed?

**Comment by emmauss** [comment]
It looks like the default style for the indicator isn't being applied. This is occurring on the samples on master.
